jobs/config.json

```diff
@@ -7298,6 +7298,7 @@
       "--gke-environment=staging",
       "--provider=gke",
       "--skew",
+      "--test_args=--ginkgo.skip=Initializers",
```

Comment:
I think the `-cluster-new` suite runs skewed tests (1.7, does not have Initializers?)
http://k8s-testgrid.appspot.com/google-gke-staging#gke-staging-1-7-1-8-upgrade-cluster-new looks healthy-ish

Reply:
Yeah, the passing tests are also just skipping the test logics based on disabled feature gates and runtime API.